lib/EasyDirectiveVisitor.ts

```diff
@@ -302,10 +309,66 @@ abstract class EasyDirectiveVisitor<
   ): DocumentNode[] {
     const unknownTypes: GraphQLNamedType[] = [];
     this.commonTypes.forEach(type =>
+      // @ts-expect-error (FIXME: type is not assignable)
       collectUnknownNamedTypes(schema, type, unknownTypes),
     );
     return unknownTypes.map(type => gql(printType(type)));
   }
+
+  /* eslint-disable class-methods-use-this, class-methods-use-this, @typescript-eslint/no-unused-vars, @typescript-eslint/no-explicit-any */
+  // istanbul ignore next (should be overridden and never reached)
+  public visitArgumentDefinition(
+    argument: GraphQLArgument,
+    { field }: { field: GraphQLField<unknown, unknown, any> },
+  ): void {
+    throw new Error('Method not implemented.');
+  }
+
+  // istanbul ignore next (should be overridden and never reached)
+  public visitInputObject(object: GraphQLInputObjectType): void {
+    throw new Error('Method not implemented.');
+  }
+
+  // istanbul ignore next (should be overridden and never reached)
+  public visitInputFieldDefinition(
+    field: GraphQLInputField,
+    { objectType }: { objectType: GraphQLInputObjectType },
+  ): void {
+    throw new Error('Method not implemented.');
+  }
+
+  // istanbul ignore next (should be overridden and never reached)
+  public visitFieldDefinition(
+    field: GraphQLFieldConfig<unknown, TContext, TArgs>,
+    { objectType }: { objectType: GraphQLObjectType<any, any> },
+  ): void {
+    throw new Error('Method not implemented.');
+  }
+
+  // istanbul ignore next (should be overridden and never reached)
+  public visitObject(
+    object: GraphQLInterfaceType | GraphQLObjectType<any, any>,
+  ): void {
+    throw new Error('Method not implemented.');
+  }
+
+  // istanbul ignore next (should be overridden and never reached)
+  public visitQuery(
+    query: GraphQLObjectType<any, TContext>,
+    schema: GraphQLSchema,
+    directiveName: string,
+  ): void {
+    throw new Error('Method not implemented.');
+  }
+  /* eslint-enable class-methods-use-this, class-methods-use-this, @typescript-eslint/no-unused-vars, @typescript-eslint/no-explicit-any */
```

Comment:
nit: 
Wouldn't it be better to use Generics to ensure the directives override the methods we want them to instead of having these methods returning 'void` ?
- ex:
```typescript
type AllowedVisitors = VisitObject | ...;
class EasyDirectiveVisitor<TOverwrittenVisitors extend DirectiveLocation> {
  ...
  // returnType = `void` when DirectiveLocation contains at least one match; Otherwise, returns `never` 
  // This implies that:
    // This function returns `never` if the `DirectiveLocation` is not informed (true)
    // This function returns `void` if `DirectiveLocation` is not informed( type error )
  public visitObject(...): TOverwrittenVisitors extend <DirectiveLocationsThatUseThisVisitor> ? void : never {
    throw new Error('Method not implemented');
  }
}

class SomeDirectiveVisitor extends EasyDirectiveVisitor<DirectiveLocations> {
  ...
  // error if not overriding visitObject, since the method in the base class returns `never`
  public visitObject(...): void {...}
}
```

Reply:
I couldn't make it work like you said.

TS raises an error if I try to override a method that returns `never` with one that returns `void` (because the type is not assignable), but it doesn't complains if I don't override it.

Reply:
changed to use generics like you said. Thanks! :D